### PR TITLE
docs: more explanation

### PR DESCRIPTION
**Pull request by aldeed**
Adds some more details to the readme docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,13 +6,29 @@
 
 ## Summary
 
-Simple Schema plugin for the [Reaction API](https://github.com/reactioncommerce/reaction).
+SimpleSchema plugin for the [Reaction API](https://github.com/reactioncommerce/reaction).
 
-This plugin adds support for putting SimpleSchemas on context, so that other plugins can extend them.
+The [SimpleSchema](https://github.com/aldeed/simple-schema-js) package is used by many plugins to validate data, often before inserting or updating MongoDB documents. To allow other plugins to extend these schemas, some plugins register them such that they are accessible in a `preStartup` function.
+
+simpl-schema
```
Comment (mpaktiti) [inline README.md]: Is this supposed to be here?

**Review by mpaktiti — APPROVED** [review]
Looks good, I added a comment on a line that seems like a copy/paste mistake, other than that good to go.

File: README.md (hunk 1 above)
Comment (aldeed) [reply README.md]: Nope!

**Pull request merged**